### DISCUSSION TITLE
Pimcore 11 PimcoreTinymce with textcolor

### DISCUSSION THREAD

**Discussion by chrobaks** [pimcore/pimcore#17506] [General]
i'm using PimcoreTinymce in Pimcore 11 and i need to edit the color of text, when i work with pimcore_wysiwyg

I try this configuration in config/config.yaml , but it don't work: 

`documents:
    richtext_editor: tinymce
        tinymce:
            plugins:
                - textcolor
                - colorpicker
                toolbar: 'textcolor | undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image'``

**Comment by muratbinerbay** [comment]
If you want to customise the TinyMCE editor for a single template then you can pass a config to `pimcore_wysiwyg`. Or if you wanted to change the config for the editor globally, do it in an `editmode.js` that gets loaded in the admin area.

Docs: https://github.com/pimcore/pimcore/blob/11.x/bundles/TinymceBundle/README.md
